assets/website/lang/webmessages_pl.properties

```diff
@@ -173,25 +173,25 @@ user_welcome = "Witaj,"
 user_vip = "Jest to konto VIP"
 
 [modpacks]
-name = "Modpacks"
+name = "Paczki modów"
 
-title = "Modpacks"
+title = "Paczki modów"
 
-notready = "Modpacks are planned for next big update"
-button_back = "Go back to homepage"
+notready = "Paczki modów zostały zaplanowane na następną dużą aktualizację"
+button_back = "Powrót do strony głównej"
 
 [register_mail]
-subject = "Register Confirmation"
+subject = "Potwierdzenie rejestracji"
 
-welcome = "Hi"
-text = "thank you for registering your account on SKplatform."
-button_confirm = "Confirm My Email Address"
+welcome = "Cześć,"
```

Comment:
Unnecessary comma as it's used like `Hi {nickname},`

Reply:
oh ok